### PR TITLE
Add v1.2.2 release note

### PR DESCRIPTION
**Pull request by bk201**

### CODE REVIEW

File: v1.2.2.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+:file_folder: https://releases.rancher.com/harvester/v1.2.2/harvester-v1.2.2-rootfs-amd64.squashfs
+
+:white_check_mark: https://releases.rancher.com/harvester/v1.2.2/harvester-v1.2.2-amd64.sha512
+
+:memo: https://releases.rancher.com/harvester/v1.2.2/version.yaml
+
+
+## Enhancements
+
```
Comment (jillian-maroket) [inline v1.2.2.md:27]:
```suggestion
## Installation

Harvester can be installed using the [ISO image](https://docs.harvesterhci.io/v1.2/install/index), a [bootable USB drive](https://docs.harvesterhci.io/v1.2/install/usb-install), and [PXE boot](https://docs.harvesterhci.io/v1.2/install/pxe-boot-install). For more information, see the [Installation](https://docs.harvesterhci.io/v1.2/install/requirements) section of the documentation.

## Upgrade

Harvester only allows upgrades from supported versions. For more information about upgrade paths and procedures, see [Upgrading Harvester](https://docs.harvesterhci.io/v1.2/upgrade/index).

> [!NOTE]
> Harvester will eventually support upgrading from v1.2.2 to v1.3.1. Once that version is released, you must first upgrade a Harvester cluster to v1.2.2 before upgrading to v1.3.1.

## Enhancements
```

File: v1.2.2.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+- [BUG] Unable to install to MBR disks [#4515](https://github.com/harvester/harvester/issues/4515)
+- [BUG] MTU is overwrite by storage network [#4297](https://github.com/harvester/harvester/issues/4297)
+- [BUG] SR-IOV Virtual Functions can be 0 or -N integers [#3977](https://github.com/harvester/harvester/issues/3977)
+- [EPIC] Add further pre and post checks in harvester upgrade [#3067](https://github.com/harvester/harvester/issues/3067)
+
+
+## Others
+
+
+## Known Issues
```
Comment (jillian-maroket) [inline v1.2.2.md:120]:
```suggestion
```

File: v1.2.2.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+- [backport v1.2] [ENHANCEMENT] GUI: change from Ntp server to NTP servers in settings [#4825](https://github.com/harvester/harvester/issues/4825)
+- [backport v1.2] [ENHANCEMENT] Check free disk space percent before upgrade [#4801](https://github.com/harvester/harvester/issues/4801)
+- [backport v1.2] [ENHANCEMENT] config sftp dynamically [#4788](https://github.com/harvester/harvester/issues/4788)
+- [backport v1.2] [FEATURE] Generate and upload code coverage report for integration tests [#4766](https://github.com/harvester/harvester/issues/4766)
+- [ENHANCEMENT] collect containerd.log in support-bundle-kit [#4427](https://github.com/harvester/harvester/issues/4427)
+- [ENHANCEMENT] Support sshd configuration [#4231](https://github.com/harvester/harvester/issues/4231)
+
+
+## Bug fixes
+
```
Comment (jillian-maroket) [inline v1.2.2.md:56]:
```suggestion
## Bug Fixes
```

**Review by jillian-maroket — COMMENTED** [review]
Please add the missing content and remove the empty sections (see my comments). Thank you.

File: v1.2.2.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+## Known Issues
+
+## Component Versions
+
+| Component | Version |
+| --- | --- |
+| Longhorn | [v1.5.5](https://github.com/longhorn/longhorn/releases/tag/v1.5.5) |
+| KubeVirt | [v1.1.1](https://github.com/kubevirt/kubevirt/releases/tag/v1.1.1) |
+| Embedded Rancher | [v2.8.2](https://github.com/rancher/rancher/releases/tag/v2.8.2) |
+| RKE2 | [v1.26.15+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.26.15%2Brke2r1) |
```
Comment (innobead) [inline v1.2.2.md:129]: As discussed, let's add SLE Micro dependency here, as Harvester is kind of a variant version of it.

**Review by innobead — COMMENTED** [review]
In general, LGTM. After fixing @jillian-maroket comments, I think we are good.

File: v1.2.2.md (hunk 4 above)
Comment (bk201) [reply v1.2.2.md:130]: Fixed

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
@bk201 Merge this when you think it's ready. Thanks.

**Pull request merged**